tests/dummy/app/app.js

```diff
@@ -2,6 +2,7 @@ import Ember from 'ember';
 import Resolver from './resolver';
 import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
+import './models/custom-inflector-rules';
```

Comment:
what is the purpose?

Reply:
Afaik we should use inflector to fix urls like 'metadata -> metadatas'. Otherwise i get the error: "XMLHttpRequest cannot load http://.../odata/NewPlatformFlexberryGISLayer**Metadata**?..."